MELA/interface/TVar.hh

```diff
@@ -182,6 +180,10 @@ namespace TVar{
     SelfDefine_spin1,
     SelfDefine_spin2,
 
+    /** Madgraph Signal, Background, BSI **/
+    Madgraph_sig,
+    Madgraph_bkg,
+    Madgraph_bsi,
```

Comment:
Why do you need separate processes for MG? Aren't HSMHiggs, bkgZZ, and bkgZZ_SMHiggs enough?

Reply:
I guess this is true - but there are some stark differences between madgraph and JHUGen/MCFM elements. For instance, everything is based off of the SMEFTsim calculation base parameters (so changing the higgs mass would change the Z or W mass, for instance), which is why I wanted to emphasize that with a different process.

If you think that the different matrix element is enough to ensure the user doesn't do something unwanted then yeah I can change it

Reply:
I think a different ME setting should be enough - people should be able to keep track of different MELA behavior from the documentation.

Reply:
Done